README.md

```diff
@@ -7,7 +7,7 @@
 
 Browser based Addons for your test driven development.
 
-Note: This branch is for CakePHP 3.5+.
+Note: This branch is for CakePHP 3.6+.
```

Comment:
Why? If we do that we would need to increase composer, etc, too.

Reply:
cake 3.5 doesn't have the `BasePlugin.php` file in the core, this feature is only available since 3.6

Reply:
But if we keep also old style both works?
I am ok with 3.6 only though. Just needs composer constraint.

Reply:
I would suggest increasing the constraints to require php 7.1 at least and cakephp to 3.6, yeah

Reply:
ok, well like i said, if you merge this you increase the requirement up to 3.6+, so you decide :)